### PR TITLE
startPage & homePage CSS, Basic Landing page

### PR DESCRIPTION
**Pull request by nickfong**
Can theoretically replace ~~#73~~ #74

### CODE REVIEW

File: app/assets/html/navigation_bar.html [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
             <a class="navbar-brand" ui-sref="home">The Colosseum</a>
         </div>
-        <div class="collapse navbar-collapse">
+        <div collapse="!showMenu" class="collapse navbar-collapse">
+            <ul class="nav navbar-nav">
+                <li>
+                    <a id="home_nav" ui-sref="home">Home</a>
```
Comment (nickfong) [inline app/assets/html/navigation_bar.html]: There shouldn't be a `Home` button anymore

File: app/assets/javascripts/homepage/conversation.coffee [hunk 2]
```diff
@@ -194,4 +194,9 @@ conversation.controller("ConversationController", ["$scope", "$stateParams", "AP
     , CONVERSATION_POLL_PERIOD * 1000)
 
     updateConversationData(true)
+    # resize = () ->
+    #     element = document.getElementById("posts-container")
+    #     element.setAttribute("style", "height:" + (window.innerHeight - 390) + "px")
+    # window.onresize = resize
+    # resize()
```
Comment (nickfong) [inline app/assets/javascripts/homepage/conversation.coffee]: Delete?

**Comment by nickfong** [comment]
+1 once my two comments are addressed

File: public/landingpage.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                width: 100%;
+                height: 100%;
+                z-index: 900;
+                opacity: 0.7;
+            }
+
+            #title {
+                margin-bottom: 30px;
+            }
+        </style>
```
Comment (leohtkam) [inline public/landingpage.html:56]: Why is this in the html?

File: public/landingpage.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    </head>
+    <body>
+        <div id="background-overlay"></div>
+        <div id="main-view-container" class="container">
+            <div id="main-view" ui-view="content">
+                <h3 id="title">PenPal Gladiators</h3>
+                <p id="tag">
+                    Politics is a complicated topic. People have disagreements and it's hard to agree on something. But what if you could talk about it? To be able to discuss differences with someone of opposing opinion?
```
Comment (nickfong) [inline public/landingpage.html:65]: We probably want to refine what we're saying here but it's good as a placeholder/for now

**Comment by leohtkam** [comment]
+1, in chromium on virtualbox I have some problem with not being able to scroll down (in edit profile and a conversation with a lot of posts), but Kevin hasn't be able to reproduce the bug so maybe it's just my VM. It works on other web browsers on my VM.

**Pull request merged**